src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -948,6 +948,36 @@ struct GaugeHImplicitFrom3p1QuantitiesCompute : GaugeH<SpatialDim, Frame>,
   using base = GaugeH<SpatialDim, Frame>;
 };
 
+/*!
```

Comment:
Do you need this for observers? If not, is it possible to implement this as only a free function, so that we can avoid adding compute tags?

Reply:
the spatial derivative of GaugeH is needed by `FConstraint` and `TwoIndexConstraint`. To observe them I need to have `::Tags::deriv<GaugeH<SpatialDim, Frame>, tmpl::size_t<SpatialDim>, Frame>` in the databox.

I can make this a free function, and modify the compute tags for the F- and 2index-constraints to take in the spacetime derivative of GaugeH (which is already available), and extract the spatial deriv from it before continuing the computation of constraints. The downside of this is only that the computation will be repeated (once for each constraint).  Is this something you'd prefer?